src/anemoi/training/train/forecaster.py

```diff
@@ -293,12 +295,28 @@ def get_variable_scaling(
             pressure_level.minimum,
         )
 
+        tendency_scaler = instantiate(config.training.tendency_scaler)
+
+        LOGGER.info(
+            "Tendency variable scaling: using scaler %s",
+            type(tendency_scaler).__name__,
+        )
+
         for key, idx in data_indices.internal_model.output.name_to_index.items():
+            if idx in self.data_indices.internal_model.output.prognostic:
+                prog_idx = self.data_indices.data.output.name_to_index[key]
+                variable_stdev = 1 / self.model.pre_processors.processors.normalizer._norm_mul[prog_idx]
+                variable_tendency_stdev = (
+                    self.statistics_tendencies["stdev"][prog_idx] if self.statistics_tendencies else 1
+                )
+                scaling = tendency_scaler.scaler(variable_stdev, variable_tendency_stdev)
+                LOGGER.debug("Parameter %(key)s is being scaled by statistic_tendencies by %(scaling).2f")
```

Comment:
This didn't work for me but the following did:

LOGGER.debug(f"Parameter {key} is being scaled by statistic_tendencies by {scaling:.2f}")